ocs_ci/ocs/fiojob.py

```diff
@@ -288,6 +288,38 @@ def delete_fio_data(fio_job_file, delete_check_func):
         raise Exception(error_msg)
 
 
+def get_sc_name(fixture_name):
+    """
+    Return storage class name based on fixture name suffix.
+    """
+    if fixture_name.endswith("rbd"):
+        if config.DEPLOYMENT.get("external_mode"):
+            storage_class_name = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_RBD
+        else:
+            storage_class_name = constants.DEFAULT_STORAGECLASS_RBD
+    elif fixture_name.endswith("cephfs"):
+        if config.DEPLOYMENT.get("external_mode"):
+            storage_class_name = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_CEPHFS
+        else:
+            storage_class_name = constants.DEFAULT_STORAGECLASS_CEPHFS
+    else:
+        raise UnexpectedVolumeType("unexpected volume type, ocs-ci code is wrong")
+    return storage_class_name
+
+
+def get_pool_name(fixture_name):
+    """
+    Return ceph pool name based on fixture name suffix.
+    """
+    if fixture_name.endswith("rbd"):
+        ceph_pool_name = "ocs-storagecluster-cephblockpool"
+    elif fixture_name.endswith("cephfs"):
+        ceph_pool_name = "ocs-storagecluster-cephfilesystem-data0"
```

Comment:
You might want to use an existing function to get cephfs data pool name present [here](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/helpers.py#L751)

Reply:
Good point.